slack-moderator/users.go

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+
+	"sigs.k8s.io/slack-infra/slack"
+)
+
+func (h *handler) getDisplayName(id string) (string, error) {
```

Comment:
non-blocking suggestion: these don't look like handler methods, they look like client methods? AFAICT the only structure in handler used is the client, perhaps break them out? /shrug

Reply:
They aren't really handler methods, but the client is defined in another package.

The good solution is probably to move these over to that client package. They mostly exist like this because they used to not really actually use it until I got tired of Slack being weird and added `CallOldMethod`.